model/cachedfeed.py

```diff
@@ -82,13 +83,15 @@ class CachedFeed(Base):
 
     @classmethod
     def fetch(cls, _db, worklist, facets, pagination, refresher_method,
-              max_age=None
+              max_age=None, raw=False, **response_kwargs
```

Comment:
A lot of these methods now take **response_kwargs. max_age is treated specially here, even though it ends up being passed in to the Response constructor, because it's _also_ used to determine the maximum age of a feed in the database cache.